message MSG_CLIENT_DM_USER_INFO_UPDATE
{
    optional string token = 1;
    optional string password = 2;
}

message MSG_DM_CLIENT_USER_INFO_UPDATE_ACK
{
    optional int32 statusCode = 1;
}